### DISCUSSION TITLE
How to do extended operations like power, sqrt and other

### DISCUSSION THREAD

**Discussion by ControlSoup** [iliekturtles/uom#442] [Q&A]
I think this is implemented but am having trouble finding it in the docs. 

How can I square, log or square-root a quantity? Are there implemented methods for doing so?

**Comment by iliekturtles** [comment]
The easiest way to find details is to search in the documentation. For example, [`sqrt`](https://docs.rs/uom/latest/uom/si/struct.Quantity.html#method.sqrt) is implemented when the underlying storage type is `f32`/`f64`. Once you find the method you may need to scroll up to see the [`impl` block](https://docs.rs/uom/latest/uom/si/struct.Quantity.html#impl-Quantity%3CD,+U,+f32%3E) for details about when the method is available.
